addon.json

```diff
@@ -0,0 +1,6 @@
+{
+    "package-name": "djangocms-audio",
+    "installed-apps": [
```

Comment:
this is actually deprecated. The preferred way of doing it is in `aldryn_config.Form#to_settings`